packages/deployment/app/cumulus_api.template.yml

```diff
@@ -166,6 +166,24 @@ Resources:
       Environment:
         Variables:
           stackName: {{../stackName}}
+          VARIABLE: variable
```

Comment:
the `VARIABLE: variable` seems to be redundant.

Reply:
Oh goodness, thanks! That was a debug thing.